src/controller.rs

```diff
@@ -87,6 +84,7 @@ impl CanRetry for SendWithRetry {
 
 pub struct MetricsController {
     is_active: bool,
+    logger: MetricsLogger,
```

Comment:
So, MetricsLogger is kept in the MetricsController, how do I use it from other modules?  Can I follow the example that gzip uses and just call the macros debug!, err! (is there an info!)? and that will get it logged to the logger?